src/app/src/components/Quiz.js

```diff
@@ -31,7 +32,8 @@ class Quiz extends React.Component {
         super();
         const answers = shuffle(QUIZ_FISH).slice(0, NUM_QUIZ_QUESTIONS);
         const choices = answers.map(fish => {
-            const otherFish = QUIZ_FISH.filter(
+            // draw answers from the full fish catalog for more variety
+            const otherFish = FISH_CATALOG.filter(
                 a => a.commonName !== fish.commonName
```

Comment:
Consider also excluding other _answers_ here, so that you don't have as a possible option a fish that was the answer to a previous question:
```suggestion
                 a => a.commonName !== fish.commonName && !answers.find(b => a.commonName === b.commonName)
```

Reply:
Is this bad though? The user doesn't know the fish couldn't show up again.

Reply:
:man_shrugging: It struck me when playing through the quiz as an obviously wrong answer (i.e. "It can't be that fish, I already picked that one and it looked different"), but I do have more context than the museum-goers will.

I could see an argument for keeping things the way they are now as well.

Reply:
Resolved to keep things as is as per @alexelash on Slack so users can build on their knowledge in the short time window of the quiz~